### ISSUE TITLE
open-vm-tools-10.1.15.tar.gz contains 10.1.10 build 6082533

### ISSUE THREAD

**Issue by ml8mr** [vmware/open-vm-tools#198]
On the releases page for this project (https://github.com/vmware/open-vm-tools/releases) there are three possible archives which should contain the latest 10.1.15 code. However, the Changelog in the official tarball (open-vm-tools-10.1.15.tar.gz) has the following first line:

    open-vm-tools-10.1.10 build 6082533

In the subsequent two source archives (both the .zip and the .tar.gz), the first line in the Changelog is:

    open-vm-tools build 6677369

In other words, the open-vm-tools-10.1.15.tar.gz archive doesn't appear to be current.

**Comment by oliverkurth** [comment]
Thanks for reporting! It looks like the *.tar.gz is missing just the last commit: https://github.com/vmware/open-vm-tools/commit/ef87eddf942bb71f1184e7ff992949653ef95fad#diff-1d0e029b56a3855fd93b81205b540dc1

I'll fix it.

**Comment by oliverkurth** [comment]
I fixed it. I also renamed the tarball to include the build number, just as we did in previous releases. This should help identify the corrected tarball.
